tests/slsa_analyzer/package_registry/test_npm_registry.py

```diff
@@ -17,14 +17,14 @@
 
 @pytest.fixture(name="npm_registry")
 def create_npm_registry() -> NPMRegistry:
-    """Create an npm registry instance."""
+    """Create an NPM registry instance."""
```

Comment:
`npm` should be lowercase. See https://docs.npmjs.com/cli/v7/using-npm/registry

Reply:
But is the comment referring to the class or the external system the class represents? I'll change it, I'm just commenting that it is a bit hard to follow their preferred naming convention when it directly conflicts with our own class naming convention and we ignore it there.

Reply:
The comment is referring to the external `npm` registry. But can I see how it gets tricky to figure out the right naming convention.